src/main/java/course/info/DataToDBLoader.java

```diff
@@ -0,0 +1,87 @@
+package course.info;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+//import java.sql.ResultSet;
+//import java.sql.Statement;
+//import java.sql.Connection;
+import java.sql.PreparedStatement;
+//import java.sql.ResultSet;
+import java.sql.SQLException;
+//import java.util.ArrayList;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import FuncTest.TestFunc.globals;
+import Utils.JsonReader;
+
+public class DataToDBLoader {
+	
+	public static void loadToDBFromJson(String jsonName) {
+		JSONArray courses = JsonReader.ReadJsonFromFile(jsonName);
+		 for (int c = 0; c < courses.length(); ++c) {
```

Comment:
Please do not insert big amounts of data into the database using multiple queries as this has a chance to temporarily block our azure account ,instead use batching as explained here: https://viralpatel.net/blogs/batch-insert-in-java-jdbc/

Reply:
@Itamaror great comment, Thanks! wil update